### DISCUSSION TITLE
Amazon大裁員

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#661] [General]
Amazon預計在2023年1月18號開始進行裁員與關店，為何要這麼做呢？
Amazon CEO Andy Jassy寫到：
“These changes will help us pursue our long-term opportunities with a stronger cost structure; however, I’m also optimistic that we’ll be inventive, resourceful, and scrappy in this time when we’re not hiring expansively and eliminating some roles.”
Amazon在疫情肆虐期間快速且大量的僱用了很多員工，光是第三個季度就僱用多達1.54million的人，因此去年11月Amazon就有消息要解僱一些人，所以各位我們一定要想變法讓自己未來穩住自己的位子，一定要握有至少一樣別人無法取代的長處啊！

文章來源：https://www.cnbc.com/2023/01/04/amazon-says-it-will-cut-over-18000-jobs-more-than-initially-planned.html

**Comment by 710388** [comment]
想要順應時代的潮流 只有提升自己的能力 才能應付各種狀況 當上董事長 就裁不掉了

**Comment by bww0401** [comment]
在這個工作隨時可能會被取代的時代，唯有增加自己的能力與獨特性，才能在職場上生存，共勉之。
